pkgs/top-level/all-packages.nix

```diff
@@ -1105,6 +1105,16 @@ in
 
   apksigcopier = callPackage ../development/tools/apksigcopier { };
 
+  apksigner =
+    let
+      jdk = jdk11;
+      build-tools = androidenv.androidPkgs_9_0.build-tools;
+    in runCommand "apksigner" { nativeBuildInputs = [ makeWrapper ]; } ''
+      mkdir -p $out/bin
+      makeWrapper "${jdk}/bin/java" "$out/bin/apksigner" \
+        --add-flags "-jar ${builtins.head build-tools}/libexec/android-sdk/build-tools/30.0.3/lib/apksigner.jar"
+    '';
```

Comment:
Please put this in its own file and make the commit message start with `apksigner: init`.

Reply:
done.